src/pages/selectGame/selectGame.js

```diff
@@ -1,28 +1,23 @@
 import React from 'react';
 import HeaderContainer from '../../containers/header';
+import GameSettings from '../../containers/gamesettings/gameSettings';
+import { Button } from '../../components';
 
 import './styles/selectGame.css';
 
 export default function SelectGame() {
   return (
     //functions
-      // header - check
+    // header - check
     //game type singles doubles
     //player types
     //scoretype
     //backbutton
 
     <div className="container">
       <HeaderContainer />
-      <div>
-        <h2>Buttons and players input</h2>
-      </div>
-      <div>
-        <h2>game settings</h2>
```

Comment:
whats about game settings?

Reply:
they are on the green... is it possible that you got confused?